contrib/devtools/security-check.py

```diff
@@ -143,6 +143,20 @@ def check_ELF_separate_code(executable):
                 return False
     return True
 
+def check_ELF_control_flow_instrumentation(executable) -> bool:
+
+    elf = pixie.load(executable)
+
+    if elf.hdr.e_machine is not pixie.EM_X86_64:
+        return True
```

Comment:
At some point we might want to add a 'skipped' return status for checks that could not be performed for a platform (they would count as true for the final return value, but might be displayed differently).

Reply:
Agree. I think printing some sort of informational output may be useful.